src/Components/Shops/CreateShop.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, Form, Input, Typography } from 'antd';
+import { CiShop } from 'react-icons/ci';
+
+const CreateShop = () => {
+  return (
```

Comment:
use 'useForm' hook to initialise the Form. and add the Datatype there which will help you to validate the dataType of the form. 

`const [form] = useForm<FormSubmitEventInterface>()`

Reply:
I don't see any changes.Can you confirm this, you have changed according to my comment?